nats.go

```diff
@@ -589,21 +589,30 @@ type srv struct {
 
 // The INFO block received from the server.
 type serverInfo struct {
-	ID           string   `json:"server_id"`
-	Host         string   `json:"host"`
-	Port         uint     `json:"port"`
-	Version      string   `json:"version"`
-	AuthRequired bool     `json:"auth_required"`
-	TLSRequired  bool     `json:"tls_required"`
-	TLSAvailable bool     `json:"tls_available"`
-	Headers      bool     `json:"headers"`
-	MaxPayload   int64    `json:"max_payload"`
-	ConnectURLs  []string `json:"connect_urls,omitempty"`
-	Proto        int      `json:"proto,omitempty"`
-	CID          uint64   `json:"client_id,omitempty"`
-	ClientIP     string   `json:"client_ip,omitempty"`
-	Nonce        string   `json:"nonce,omitempty"`
-	LameDuckMode bool     `json:"ldm,omitempty"`
+	AuthRequired      bool     `json:"auth_required,omitempty"`
+	CID               uint64   `json:"client_id,omitempty"`
+	ClientConnectURLs []string `json:"connect_urls,omitempty"` // Contains URLs a client can connect to.
+	ClientIP          string   `json:"client_ip,omitempty"`
+	Cluster           string   `json:"cluster,omitempty"`
+	Dynamic           bool     `json:"cluster_dynamic,omitempty"`
+	GitCommit         string   `json:"git_commit,omitempty"`
+	GoVersion         string   `json:"go"`
+	Headers           bool     `json:"headers"`
+	Host              string   `json:"host"`
+	ID                string   `json:"server_id"`
+	IP                string   `json:"ip,omitempty"`
+	JetStream         bool     `json:"jetstream,omitempty"`
```

Comment:
Would remove JetStream and LDM since we already process that IIRC.

Reply:
We dont process JetStream and LDM was here already - I considered adding a function to expose the JetStream property but I dont think it's useful, in typical real world clusters the individual servers might not be JS enabled only the storage node etc, do you think exposing it will be useful?

Reply:
Correct, would remove JS for now..